### ISSUE TITLE
"The context paramter [sic] is invalid" error if Experimental Web Platform features flag is on

### ISSUE THREAD

**Issue by anssiko** [webmachinelearning/webnn-polyfill#186]
All samples at https://webmachinelearning.github.io/webnn-samples/ fail with:

>The context paramter is invalid

If `chrome://flags/#enable-experimental-web-platform-features` is enabled.

Tested with Chrome Stable 107 and Canary 109.

**Comment by BruceDai** [comment]
Yes, when testing by Chrome with enabled **enable-experimental-web-platform-features** flag, the following code would invoke `createContext()` function of ML interface from Model Load API.

```js
const context = navigator.ml.createContext(contextOptions);
```

The [`createContext()` function of Model Load API](https://webmachinelearning.github.io/model-loader/#dom-ml-createcontext) is an async function, it returns a **Promise\<MLContext\>** object, while WebNN API [MLGraphBuilder constructor](https://webmachinelearning.github.io/webnn/#dom-mlgraphbuilder-mlgraphbuilder) requires a context parameter of **MLContext** type, so the error was alerted. 
```js
this.builder_ = new MLGraphBuilder(context);
```
To fix this issue need modify code using `await`, @Honry PTAL, thanks.
```js
const context = await navigator.ml.createContext(contextOptions);
```
while there's an issue https://github.com/webmachinelearning/model-loader/issues/41 of Model Load API.

And current WebNN Polyfill API checks whether Navigator object has ML interface, if not, WebNN Polyfill API will provide it.
```ts
if (navigator.ml == null) {
  navigator.ml = new ML();
}
```

**Comment by Honry** [comment]
We are enabling latest async and sync APIs (PRs: https://github.com/webmachinelearning/webnn-polyfill/pull/172, https://github.com/webmachinelearning/webnn-samples/pull/150), which should fix this issue.
